portfolio/src/main/webapp/index.html

```diff
@@ -161,17 +161,26 @@ <h1>Contact Me</h1>
           <p>
               <input id="submit" type="submit" />
           </p>
+          </form>
           <p>
-              <label for="maxcomments">Choose the Number of Comments to Display Below:</label>
-              <select name="maxcomments" id="maxcomments" onchange="getComments(this.value);">
-                  <option value="5" selected>5</option>
-                  <option value="10">10</option>
-                  <option value="15">15</option>
-                  <option value="30">30</option>
-                  <option value="50">50</option>
-              </select>
+              <div id="display-options">
+                  <div id="maxcomments-div">
+                      <label for="maxcomments">Choose the Number of Comments to Display Below:</label>
+                      <select name="maxcomments" id="maxcomments" onchange="getComments(this.value, document.getElementById('filter-input').dataset.filterInput);">
```

Comment:
When the `onChange` statement gets this long, I would probably move it to a short function in your JS file and reference that. You can find out what the expected function looks like here: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onchange

Reply:
Got it, will change this!